src/Schema/Events/index.ts

```diff
@@ -1573,6 +1575,10 @@ export enum ActionType {
    * Corresponds to {@link TooltipViewed}
    */
   tooltipViewed = "tooltipViewed",
+  /**
+   * Corresponds to {@link ProgressiveOnboardingTooltipViewed}
+   */
+  progressiveOnboardingTooltipViewed = "progressiveOnboardingTooltipViewed",
```

Comment:
Is this used anywhere? [ProgressiveOnboardingTooltipViewed](https://github.com/artsy/cohesion/pull/566/files#diff-2181c09c8693ccb9a0667bf93e9caf26546fa65a0f99a0002aca0e0dd44e8ad8R100) uses `tooltipViewed`.

Reply:
omg i got confused with the setup. correct, it should not be there